src/varpubs/pubmed_db.py

```diff
@@ -1,37 +1,156 @@
 from dataclasses import dataclass, field
 from pathlib import Path
+from typing import Optional, Iterable
 import sqlalchemy
-from sqlmodel import Field, SQLModel
-
+from sqlmodel import Field, SQLModel, Session, Integer
+from varpubs.hgvs_extractor import extract_hgvsp_from_vcf
+from Bio import Entrez
 
+# New table: PMIDs -> Article metadata
 class PubmedArticle(SQLModel, table=True):
-    pmid: int = Field(primary_key=True)
+    pmid: int = Field(Integer, primary_key=True, nullable=False)
     title: str
     abstract: str
     authors: str
     journal: str
     pub_date: str
     doi: str
 
+# New table: term + pmid mapping
+class TermToPMID(SQLModel, table=True):
+    term: str = Field(nullable=False)
+    pmid: int = Field(foreign_key="pubmedarticle.pmid", nullable=False)
+
+    __table_args__ = (sqlalchemy.PrimaryKeyConstraint("term", "pmid"),)
+
 
 @dataclass
 class PubmedDB:
     path: Path
-    _engine: sqlalchemy.engine.base.Engine | None = field(init=False)
+    vcf_paths: Iterable[Path]
+    email: str
+    _engine: Optional[sqlalchemy.engine.base.Engine] = field(init=False, default=None)
 
     def deploy(self) -> None:
-        # wipe existing database
-        if self.path.exists():
-            self.path.unlink()
-        # create parent directories if they don't exist
-        if self.path.parent != Path():
-            self.path.parent.mkdir(parents=True, exist_ok=True)
-        # create the database
-        SQLModel.metadata.create_all(self.engine)
-
-        # load the data and insert it into the database
-        # with Session(self.engine) as session:
-        #     ...
+        print(f"[INFO] Deploying database at: {self.path}")
+        print(f"[INFO] Input VCF paths: {[str(p) for p in self.vcf_paths]}")
+        print(f"[INFO] Entrez email: {self.email}")
```

Comment:
Rather use the proper python [logging](https://docs.python.org/3/library/logging.html) module. Also consider adding verbosity flags like `-v` / `-vv` / `-vvv` to the CLI to allow users to control the logging level (e.g., `WARNING`, `INFO`, `DEBUG`) - this is a common and helpful practice in CLI tools.